package.json

```diff
@@ -72,6 +71,7 @@
     "@metamask/snaps-controllers": "^9.2.0",
     "@metamask/snaps-execution-environments": "^6.7.2",
```

Comment:
Is this package necessary? I don't see it in our source code?

Reply:
Yep removing this. This will still be included through other package peer deps, but we don't strictly need it in the `package.json`